bool x_moveForward
bool x_moveBackwards
bool x_moveLeft
bool x_moveRight
uint8 i_targetSpeed
